lib/broccoli/angular2-app.js

```diff
@@ -55,7 +56,13 @@ Angular2App.prototype.toTree = function () {
     //     outputFile: '/app.js'
     // });
 
-    return mergeTrees([assetTree, tsSrcTree, tsTree, jsTree, this.index(), vendorJsTree], { overwrite: true });
+    var lessTree = new Funnel(sourceTree, {
+        include: ['**/*.less'],
+        allowEmpty: true
+    });
+    var lessSrcTree = compileLess(lessTree);
+
+    return mergeTrees([assetTree, tsSrcTree, tsTree, jsTree, this.index(), lessSrcTree, vendorJsTree], { overwrite: true });
```

Comment:
I'm not sure how the `Funnel` works, but I think that this should also be based on `options.style`, so if the user decides to go with Sass for example it wouldn't break.

Instead of `lessSrcTree` I'd call it `stylesSrcTree`.